WhyNot/WhyNot/UNO-102/PeopleViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  PeopleImagesData.swift
+//  WhyNot
+//
+//  Created by Fernando Santos on 03/11/22.
+//
+
+import Foundation
+import SwiftUI
+
+class PeopleVM: ObservableObject {
```

Comment:
podia seguir o padrão de nome da view, e chamar PeopleImagesViewModel

```suggestion
class PeopleImagesViewModel: ObservableObject {
```